packages/manager/modules/bm-server-components/src/technical-details/technical-details.html

```diff
@@ -14,15 +22,25 @@
         term="{{:: 'dedicated_server_dashboard_technical_details_ram' | translate }}"
     >
         <oui-tile-description>
-            <span data-ng-bind=":: $ctrl.formattedRAM"></span>
+            <p data-ng-bind=":: $ctrl.formattedRAM"></p>
+        </oui-tile-description>
+        <oui-tile-description>
+            <span
+                data-ng-if="$ctrl.upgradeTask.isUpgradeScheduled($ctrl.UPGRADE_TYPE.RAM)"
```

Comment:
The `ng-if` should be applied to the whole `oui-tile-description`, to avoid empty element

Reply:
I fixup this, I fixed for storage upgrade too.